e2e_test/step_definitions/note.ts

```diff
@@ -62,6 +62,20 @@ Given(
   },
 )
 
+When(
+  "I delete the question {string} in the note {string}",
+  (questionTitle: string, noteTopic: string) => {
+    start.jumpToNotePage(noteTopic).deleteQuestion(questionTitle)
+  }
+)
+
+Then(
+  "I should not see the question {string} in the question list of the note {string}",
+  (questionTitle: string, noteTopic: string) => {
+    start.jumpToNotePage(noteTopic).expectNoQuestion(questionTitle)
+  },
```

Comment:
Checking if something "does not exist" in an async test case is not easy. Can you think of a better way than this?

Reply:
Thanks! I think I will just make sure that there is only one question left in the list after delete one.